store/src/commonMain/kotlin/com/dropbox/external/store5/operator/ShareableMutableMapOf.kt

```diff
@@ -0,0 +1,7 @@
+package com.dropbox.external.store5.operator
+
+import co.touchlab.stately.isolate.IsolateState
+import com.dropbox.external.store5.definition.ShareableMutableMap
+
+internal inline fun <Key : Any, Value : Any> shareableMutableMapOf():
+    ShareableMutableMap<Key, Value> = IsolateState { mutableMapOf() }
```

Comment:
This is just a matter of preference. But I'd probably put this in the same file as `ShareableMutableMap`, which is more aligned with the way the Kotlin APIs are structured. But your call :)